CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing
+
+## Reporting Bugs and Creating Issues
```

Comment:
I created a issue template for this today, so maybe not needed anymore or mention it here pls

Reply:
I copied / adapted the text from Clarity (which has also issues templates)